.changes/v1.0.0/81-notes.md

```diff
@@ -0,0 +1 @@
+* Add examples of how to use the VMware Cloud Foundation Automation Terraform Provider, that can be found [here](examples) [GH-81]
```

Comment:
The path is to `examples` and not `../../examples` as this will be in root CHANGELOG.md